backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/entities/User.java

```diff
@@ -54,4 +56,47 @@ public void onPersistAndUpdate() {
   private void hashPassword() {
     this.password = BCrypt.hashpw(this.password, BCrypt.gensalt(12));
   }
+
+  @Override
+  public Collection<? extends GrantedAuthority> getAuthorities() {
+    List<SimpleGrantedAuthority> authorities = new ArrayList<>();
+
+    // Add ROLE_ prefix to roles
+    if (role == UserRole.ADMIN) {
+      authorities.add(new SimpleGrantedAuthority("ROLE_ADMIN"));
```

Comment:
can you merge develop into your branch and check does /resources/db/data.sql needs to be adjusted for this change?

Reply:
It does not need to be adjusted because I did not modify the entity mapping, it's just a thing to make my User class work with Spring Security